.github/workflows/ci.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches-ignore:
       - main
       - dependabot/**
+      - backport/**
```

Comment:
We're going to skip CI for backport branches?

Reply:
CI runs on the PR to release branch, it doesn't need to also run in backport branches ;)

Reply:
Does mean that we can merge stuff that breaks the release branch, but I guess it's a valid tradeoff for delivering faster.